src/einsteinpy/symbolic/ricci.py

```diff
@@ -1,5 +1,91 @@
+import numpy as np
+import sympy
+
 from .tensor import Tensor
+from .metric import MetricTensor
 
 
 class RicciTensor(Tensor):
-    pass
+    """
+    Class for defining Ricci Tensor
+    """
+
+    def __init__(self, arr, syms):
+        """
+        Constructor and Initializer
+
+        Parameters
+        ----------
+        arr : ~sympy.tensor.array.dense_ndim_array.ImmutableDenseNDimArray or list
+            Sympy Array or multi-dimensional list containing Sympy Expressions
+        syms : tuple or list
+            Tuple of crucial symbols dentoting time-axis, 1st, 2nd, and 3rd axis (t,x1,x2,x3)
+
+        Raises
+        ------
+        TypeError
+            Raised when arr is not a list or sympy Array
+        TypeError
+            syms is not a list or tuple
+
+        """
+        super(RicciTensor, self).__init__(arr)
+        if isinstance(syms, (list, tuple)):
+            self.syms = syms
+            self.dims = len(self.syms)
+        else:
+            raise TypeError("syms should be a list or tuple")
+
+    @classmethod
+    def from_christoffels(cls, chris):
+        """
+        Get Ricci Tensor calculated from a Christoffel Symbols
+
+        Parameters
+        ----------
+        chris : ~einsteinpy.symbolic.ricci.RicciTensor
+            Christoffel Symbols from which Ricci Tensor to be calculated
+
+        """
+        arr, syms = chris.tensor(), chris.symbols()
+        dims = len(syms)
+        ricci_list = (np.zeros(shape=(dims, dims, dims, dims), dtype=int)).tolist()
```

Comment:
Ricci Tensor is 2nd order tensor
it should be `(np.zeros(shape=(dims, dims), dtype=int)).tolist()`